### ISSUE TITLE
form elements must have a visible label

### ISSUE THREAD

**Issue by tassoman** [vuejs/vitepress#2080]
### Describe the bug

Running firefox's accessibility tests fails on "toggle dark mode" switch.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#form_elements_should_have_a_visible_text_label

a11y accessibility wcag

### Reproduction

- build a basic website
- inspect it with the browser's console accessibility tool

### Expected behavior

all form input elements must have a label or attribute `aria-labelled=` with accessible value

### System Info

```shell
System:
    OS: Linux 5.10 Ubuntu 22.04.2 LTS 22.04.2 LTS (Jammy Jellyfish)
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Memory: 11.91 GB / 12.31 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v16.15.1/bin/yarn
    npm: 8.19.2 - ~/.nvm/versions/node/v16.15.1/bin/npm
  npmPackages:
    vitepress: ^1.0.0-alpha.49 => 1.0.0-alpha.49
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by yyx990803** [comment]
closed via 3b6a6d1
